content/pt-br/docs/tasks/administer-cluster/kubelet-credential-provider.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Configurar um provedor de credenciais de imagem kubelet.
+reviewers:
+- liggitt
+- cheftako
+description: Configure o plugin de provedor de credenciais de imagem do kubelet.
+content_type: task
+min-kubernetes-server-version: v1.26
+weight: 120
+---
+
+{{< feature-state for_k8s_version="v1.26" state="stable" >}}
+
+<!-- overview -->
+
+A partir do Kubernetes v1.20, o kubelet pode recuperar dinamicamente as credenciais para um registro de imagem de contêiner usando plugins executáveis. O kubelet e o plugin de execução se comunicam por meio de stdio (stdin, stdout e stderr) usando APIs versionadas do Kubernetes. Esses plugins permitem que o kubelet solicite credenciais para um registro de contêiner dinamicamente, em vez de armazenar credenciais estáticas no disco. Por exemplo, o plugin pode se comunicar com um servidor de metadados local para recuperar credenciais de curta duração para uma imagem que está sendo puxada pelo kubelet.
+
+Você pode estar interessado em usar essa capacidade se alguma das condições abaixo for verdadeira:
+
+* Chamadas de API para um serviço de provedor de nuvem são necessárias para recuperar informações de autenticação para um registro.
+* As credenciais têm tempos de expiração curtos e é necessário solicitar novas credenciais com frequência.
+* Armazenar credenciais de registro no disco ou em `imagePullSecrets` não é aceitável.
+
+Este guia demonstra como configurar o mecanismo de plugin do provedor de credenciais de imagem do kubelet.
+
+## {{% heading "prerequisites" %}}
+
+* Você precisa de um cluster Kubernetes com nós que suportem plugins de provedor de credenciais do kubelet. Esse suporte está disponível no Kubernetes {{< skew currentVersion >}}; As versões v1.24 e v1.25 do Kubernetes incluíram isso como um recurso beta, ativado por padrão.
+
+* Uma implementação funcional de um plugin exec de provedor de credenciais. Você pode construir seu próprio plugin ou usar um fornecido por provedores de nuvem.
```

Comment:
```suggestion
* Uma implementação funcional de um plugin executável de provedor de credenciais. Você pode criar seu próprio plugin ou usar um fornecido por provedores de nuvem.
```
You used "plugin executável" before. Let's keep it consistent throughout the page.